test/compute/2019-12-01/armcompute/zz_generated_dedicatedhostgroups.go

```diff
@@ -18,9 +18,8 @@ import (
 
 // DedicatedHostGroupsOperations contains the methods for the DedicatedHostGroups group.
 type DedicatedHostGroupsOperations interface {
-	// CreateOrUpdate - Create or update a dedicated host group. For details of Dedicated Host and Dedicated Host Groups please see [Dedicated Host Documentation]
-	// (https://go.microsoft.com/fwlink/?linkid=2082596
-	// [https://go.microsoft.com/fwlink/?linkid=2082596][https://go.microsoft.com/fwlink/?linkid=2082596 [https://go.microsoft.com/fwlink/?linkid=2082596]])
+	// CreateOrUpdate - Create or update a dedicated host group. For details of Dedicated Host and Dedicated Host Groups please see Dedicated Host Documentation
+	// [https://go.microsoft.com/fwlink/?linkid=2082596]
```

Comment:
Are the `[]` supposed to be there?  I don't think it's a big deal, just curious.

Reply:
Yes that's how the links are defined if they were specified as something like `[link name](https://link.com)`